### ISSUE TITLE
add usage tip: How to validate an Endevor URL

### ISSUE THREAD

**Issue by Alexandru-Dumitru** [eclipse/che-che4z-explorer-for-endevor#94]
In any browser, it's possible to access: http(s)://host:port/basePath (example: http://192.178.0.1:111/EndevorService/rest) and get a list of configurations available for that Endevor Web Service.

Output should be something like:
```
[{
  "description" : "Endevor Web Service",
  "message" : "ENDEVOR Web Service",
  "name" : "ENDEVOR",
  "status" : "Available",
  "jobName" : "JOBNAME",
  "programName" : "PROGNAME",
  "hostName" : "HOSTNAME",
  "comments" : "Access to ENDEVOR Instance",
  "userId" : "USERID",
  "password" : null,
  "poolInitSize" : 0,
  "poolIncrSize" : 0,
  "poolMaxSize" : 0,
  "poolReapTime" : 0,
  "unusedTimeout" : 0,
  "agedTimeout" : 0,
  "lang" : "EN",
  "timeZone" : "GMT",
  "characterSet" : "ISO8859-1",
  "codepage" : "cp1047",
  "contentType" : "Text",
  "traced" : "ALL"
}]
```

This can be used as a fast way to validate the URL outside of Explorer for Endevor or Zowe CLI with Endevor plugin

**Comment by jellypuno** [comment]
It would be great to have an icon that tells users the their profile is valid or not. Just a consideration

**Comment by nickImbirev** [comment]
partially implemented within #261

**Comment by roman-kupriyanov** [comment]
Since this one partially fixed in the 1.1.1 (Endevor GetInstances request is used for connection testing), I created a separate issue for the suggestion about using more icons here: https://github.com/eclipse/che-che4z-explorer-for-endevor/issues/266.
